api/src/controllers/users.js

```diff
@@ -204,7 +204,7 @@ module.exports = {
   },
   delete: (req, res) => {
     auth
-      .check(req, 'can_delete_users')
+      .check(req, ['can_edit', 'can_delete_users'])
```

Comment:
Change to `hasAllPermissions` (as above).

Reply:
`check` was refactored